packages.yml

```diff
@@ -0,0 +1,5 @@
+packages:
+  - package: dbt-labs/dbt_utils
+    version: [">=0.7.0", "<0.9.0"]
+  - package: snowplow/snowplow_web
+    version: [">=0.5.0", "<0.6.0"]
```

Comment:
You can bump the web version to `0.6.0` if it's still compatible with everything